spec/factories/users.rb

```diff
@@ -1,6 +1,14 @@
 FactoryGirl.define do
   factory :user do
-    sequence(:email_address, User.count) {|n| "liono#{n}@some-cartoon-show.com" }
+    # don't freak out if the database isn't created yet
+    # http://stackoverflow.com/a/27745684/358804
```

Comment:
Pretty sure I added that "starting value" because I had run `rake populate:*` before, but when I re-ran it, it tried to start the sequence from `0`, which threw a validation error because that email was already taken.